bundle/manifests/ptp.openshift.io_ptpoperatorconfigs.yaml

```diff
@@ -14,7 +14,28 @@ spec:
     singular: ptpoperatorconfig
   scope: Namespaced
   versions:
-  - name: v1
+  - additionalPrinterColumns:
```

Comment:
there is a reason to print all of this when the use do `oc get ptpoperatorconfig` ?

maybe just the even enabled status

Reply:
ok, I will just print the event enabled here.

Reply:
please take a look on the api change request I don't think we need the event enabled at all